lib/tasks/migrate.rake

```diff
@@ -0,0 +1,77 @@
+def reload_application_instances_model
+  # We must do this because of how the ApplicationInstance model is
+  # dynamically defined
+  ApplicationInstance.reset_column_information
+  Object.send(:remove_const, "ApplicationInstance")
+  load "app/models/application_instance.rb"
+end
+
+def reload_authentication_model
+  # We must do this because of how the Authentication model is
+  # dynamically defined
+  Authentication.reset_column_information
```

Comment:
I'm curious why this is needed.  Doesn't the rake task just end immediately after these run?